src/main/resources/data-repository-openapi.yaml

```diff
@@ -99,6 +99,26 @@ paths:
             items:
               $ref: '#/definitions/DatasetModel'
 
+  '/api/access/v1/study':
+    get:
+      description: |
+        Returns a list of all of the studies the caller has access to
+      operationId: enumerateStudies
+      tags:
+      - access
+      parameters:
+      - in: query
```

Comment:
The query parameter is speculative. I'd suggest removing it for now. We need to develop an overall approach to study and dataset search.

I'd suggest removing the query parameter from the dataset API also. (I know, I was the one who put it there...)